Sources/Datasets/datasets/image/MNIST.swift

```diff
@@ -66,6 +82,12 @@ public struct MNISTDataLoader: S5TFDataLoader {
                                       semaphore.signal()
                                     }
                 semaphore.wait()
+                do {
+                    try S5TFUtils.shell(URL(string: binaryLocation + "gunzip")!, ".s5tf-datasets/mnist/mnist_train_labels")
+                }
+                catch {
+                    fatalError("Extract command not executed.")
+                }
```

Comment:
`catch` should be on the same line as `}`. See [here](https://google.github.io/swift/#braces).